### DISCUSSION TITLE
Can I deploy discourse domcloud?

### DISCUSSION THREAD

**Discussion by sxadxsx** [domcloud/domcloud#29] [General]
Can I deploy discourse domcloud?

**Comment by willnode** [comment]
You mean discord bot? Sure, why not.

**Reply by umutcandev** [reply]
It's not Discord, it's Discourse. Discourse is forum software.

https://www.discourse.org/

**Reply by willnode** [reply]
Sorry, my bad.

I tried to install discourse without docker (has to work without docker) a long time ago. I don't remember if it success or not. Maybe you can, but not optimally. Discourse requires to be run 24/7 with Redis, which is not what we support.
